app/models/article.js

```diff
@@ -0,0 +1,16 @@
+import DS from "ember-data";
+
+const { attr, belongsTo } = DS;
+
+export default DS.Model.extend({
+    title: attr(),
```

Comment:
I don't think it's worthwhile at the moment. It _is_ nice, but IMO only useful if you're writing a library where you need a documentation site. I don't know if that's really useful for a site like this. 

Either way, we can add the comments where it makes sense and, if we decide to generate the docs later, do so.